nixos/modules/services/openmesh/xnode/admin.nix

```diff
@@ -40,14 +40,18 @@ in {
   config = lib.mkIf cfg.enable {
     environment.systemPackages = [ cfg.package ];
 
+    environment.variables = {
```

Comment:
You should likely set these in the systemd service unit file instead of globally. - https://search.nixos.org/options?channel=unstable&show=systemd.services.%3Cname%3E.environment&from=0&size=50&sort=relevance&type=packages&query=systemd.services

Reply:
You're right, I'll add it to the systemd.